mergify_engine/worker.py

```diff
@@ -1041,6 +1042,7 @@ async def start(self) -> None:
         self._redis_stream = utils.create_yaaredis_for_stream()
         self._redis_cache = utils.create_yaaredis_for_cache()
 
+        await redis_utils.load_scripts(self._redis_stream)
```

Comment:
What about doing this in `create_yaaredis_for_stream`?

Reply:
I wanted to avoid running that il all utilities `mergify-debug/mergify-refresher/mergify-worker-status/...`